### PR TITLE
https://issues.redhat.com/browse/ACM-11340 global KlusterletConfig docs

### PR DESCRIPTION
**Pull request by oafischer**
Train 15, 2.11 only

### CODE REVIEW

File: clusters/cluster_lifecycle/adv_config_cluster.adoc [hunk 1]
```diff
@@ -261,8 +264,74 @@ spec:
...
+—
+apiVersion: config.open-cluster-management.io/v1alpha1
+kind: KlusterletConfig
+metadata:
+  name: global
+spec:
+  hubKubeAPIServerURL: "example.global.com"
+----
+
+If you create another `KlusterletConfig in an environment that has a global `KlusterletConfig` and bind it with a managed cluster, the value of the global `KlusterletConfig` is used if the same field is missing or does not have a value in your `KlusterletConfig`.
```
Comment (jc-berger) [inline clusters/cluster_lifecycle/adv_config_cluster.adoc]: highlighting seems off here, I think what you mean is this: 

```suggestion
If you create another `KlusterletConfig` in an environment that has a global `KlusterletConfig` and bind it with a managed cluster, the value of the global `KlusterletConfig` is used if the same field is missing or does not have a value in your `KlusterletConfig`.
```

File: clusters/release_notes/whats_new.adoc [hunk 2]
```diff
@@ -18,6 +18,8 @@ Learn about what's new relating to Cluster lifecycle with {mce-short}.
 
 You can now push custom labels to the automation job pod created by the `ClusterCurator`. See xref:../cluster_lifecycle/ansible_config_cluster.adoc#push-cl-cr-job-pod[Pushing custom labels from the _ClusterCurator_ resource to the automation job pod] to learn more.
 
+You can now automatically apply configurations in the global `KlusterletConfig` on every managed cluster, without binding a managed cluster to a `KlusterletConfig` with an annotation. See xref:../cluster_lifecycle/adv_config_cluster.adoc#config-gloabl-klusterletconfig[Configuring the global _KlusterletConfig_] to learn more.
```
Comment (jc-berger) [inline clusters/release_notes/whats_new.adoc]: Typically, we want to start with reason before action. So, To learn more, go here. 

I think ending a sentence with a link helps clickability and usability. 

```suggestion
You can now automatically apply configurations in the global `KlusterletConfig` on every managed cluster, without binding a managed cluster to a `KlusterletConfig` with an annotation. To learn more, see, xref:../cluster_lifecycle/adv_config_cluster.adoc#config-gloabl-klusterletconfig[Configuring the global _KlusterletConfig_].
```

File: clusters/cluster_lifecycle/adv_config_cluster.adoc (hunk 1 above)
Comment (oafischer) [reply clusters/cluster_lifecycle/adv_config_cluster.adoc]: Good catch!

**Review by jc-berger — APPROVED** [review]
/lgtm

**Pull request merged**